cloudformation/template.yml

```diff
@@ -75,12 +71,25 @@ Parameters:
   Project:
     Type: String
     Default: Raster Vision CloudFormation
-    Description: Name of the project associated with the Batch job
+    Description: Name of the project to associate with new resources
```

Comment:
You might be able to make this more clear by saying "Name of the CloudFormation project to associate with new resources." 

Also, is it possible to just generate this project name? Is this something the user would care about?

Reply:
Agreed about your description, I like that phrasing better.

I think we probably could get away with generating the project name dynamically, but it's in here because `Project` and `Environment` seem to be standard variables for us in Terraform modules. If it seems like we're using it differently enough in this context that the reasoning behind those variables doesn't apply, I'd be comfortable removing it.

Reply:
If the goal of this project is to make Raster Vision accessible to those who don't have as much AWS knowledge, it seems like we could simplify things.

Reply:
True, although the project also has the goal of being useful to Azavea RV users, which might suggest that we should keep it consistent with our other modules. Opened up #17 to track discussion, since I think the decision is bigger than this PR.

Reply:
Would `Project` and `Environment` serve any purpose other than a tag? I can't see an area where we'd need a custom project name that the prefix can't provide. Also, I don't think this tool is supposed to be used outside of R&D efforts, so there wouldn't be a need to define any sort of environment, as this isn't going to be an application we're supporting.

Reply:
I think that the value of `Environment` goes down a bit within the context of how most people (internal or external) would use this template.

The value of `Prefix` is pretty clear (namespacing resources, for example, with `Prefix: Hector`). I also see value in `Project` because I might be working on two different projects under the same `Prefix` (e.g., `Project: Acme`, `Project: E Corp`).

I think you could overload `Prefix` to solve for the multi project case, but it seems more straightforward to have them separate.

Reply:
An interesting snag here is that since `Project` is only used in tags and descriptions, `Prefix` ultimately controls the namespacing of resources -- meaning that you couldn't have two `Projects` under the same `Prefix` because the names of the AWS resources would be identical. (My understanding is that at least some Batch resources need unique names -- I know this is true for [job definitions](https://docs.aws.amazon.com/batch/latest/userguide/create-job-definition.html), for example -- but I haven't confirmed it for all resources in this template). We can't rely on using the value of `Project` for namespacing because CloudFormation doesn't support string functions and it doesn't make much sense to enforce all lower-case project names. So in the situation you describe, I think you'd have to have separate prefixes for both the `Acme` and the `E Corp` project.

Reply:
In that case, I think overloading `Prefix` (and dropping the others) seems like most straightforward approach.

Reply:
Awesome, removed `Project` and `Environment`. Thanks all for sticking with this discussion and making strong cases, I think this is the right move now.